DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetOccupationsWithMatchingSkillsDetailed.json

```diff
@@ -0,0 +1,13 @@
+  "queryParams": [
+    {
+      "name": "skillList",
+      "type": "System.String[]"
```

Comment:
String is the default param type, so no need to explicitly specify it in the config (although it won't do any harm).